src/ports/xtensa_esp32/peripherals.rs

```diff
@@ -0,0 +1,15 @@
+use esp_hal::{peripherals::*, prelude::*};
+
+pub static mut PERIFERALS_RNG: Option<RNG> = None;
+pub static mut PERIFERALS_RADIO_CLK: Option<RADIO_CLK> = None;
+pub static mut PERIFERALS_WIFI: Option<WIFI> = None;
```

Comment:
Create interface for whole peripherals. For example, for uart I'll need other options of periferals, how can I get it?